src/Integrations/Integrations/IntegrationsLoader.php

```diff
@@ -77,6 +77,8 @@ public function __construct(array $integrations)
 
         // Add integrations as they support PHP 8
         if (\PHP_MAJOR_VERSION >= 8) {
+            $this->integrations[LumenIntegration::NAME] =
+                '\DDTrace\Integrations\Lumen\LumenIntegration';
```

Comment:
Is this needed? Isn't that already covered by:
https://github.com/DataDog/dd-trace-php/blob/6370bf22b1b1a93ef1830af1a167e9bd3a9226da/ext/integrations/integrations.c#L166-L167